docker-compose.yaml

```diff
@@ -0,0 +1,28 @@
+version: "3.1"
+
+services:
+  # https://github.com/suzel/docker-postgrest
+  postgrest:
+    image: suzel/docker-postgrest
+    ports:
+      - "3000:3000"
+    environment:
+      PGRST_DB_URI: postgres://app_user:secret@postgres:5432/app_db
+      PGRST_DB_SCHEMA: api
+      PGRST_DB_ANON_ROLE: web_anon
+      PGRST_JWT_SECRET: ${PGRST_JWT_SECRET}
+    links:
+      - postgres:postgres
+
+  postgres:
+    image: postgres:12.3
```

Comment:
Would stick with `12` for automatic minor version updates

Reply:
✔️